opentelemetry/src/global/logs.rs

```diff
@@ -25,6 +26,9 @@ pub trait ObjectSafeLoggerProvider {
         &self,
         library: Arc<InstrumentationLibrary>,
     ) -> Box<dyn Logger + Send + Sync + 'static>;
+
+    /// shutdown the logger provider, logs emitted after this will not be processed.
+    fn shutdown(&self) -> Vec<LogResult<()>>;
```

Comment:
why is this added? I don't think we should have explicit shutdown method at the API.

Reply:
This was added to support the global module shutdown behavior